src/components/LanguagePicker/index.js

```diff
@@ -32,6 +40,7 @@ const LanguagePicker = ({
         value={value}
         onChange={selectOnChange}
         className={className}
+        style={fontSizeStyle }
```

Comment:
When selecting "Português (BR)" as the language, the font size of the options in the LanguagePicker needs to decrease, because "Português (BR)" is a long text.
This tweak avoids this problem:

https://user-images.githubusercontent.com/614145/222805766-05b823ed-425b-4d67-9e8c-0e7538097523.mp4



And works like this:

https://user-images.githubusercontent.com/614145/222805869-86cde56e-2aa8-437f-8caf-53fc5cd5c8dc.mp4